packages/common/src/ui/forms/Detail/Section.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+import { Box, Grid, Typography } from '@mui/material';
+
+interface DetailSectionProps {
+  title: string;
+}
+export const DetailSection: FC<DetailSectionProps> = ({ children, title }) => (
+  <Grid container flex={1} flexDirection="column" gap={1}>
+    <Grid item display="flex">
+      <Box flex={1} flexBasis="40%"></Box>
+      <Box flex={1} flexBasis="60%">
+        <Typography fontWeight={700} fontSize={16}>
```

Comment:
Be good to standardise font-sizes etc. into the theme. Conversation for another day perhaps, just putting it out there for now.

Reply:
yeah. we did have an issue for font sizes; and maybe should use relative sizes than pixels